portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,33 +30,33 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.FetchOptions.Builder;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   private int numComments = 10;
+  private int offset = 0;
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
       Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
       DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
       PreparedQuery results = datastore.prepare(query);
+      FetchOptions options = FetchOptions.Builder.withLimit(numComments).offset(offset);
```

Comment:
offset should probably be a value passed in via the request but defaulted to 0 if not in the request. Like `GET /data?offset=10` or something.

Reply:
I would add that when I fetch in the javascript right? That's what I'm doing right now but for some reason it's not working. `const responsePromise = fetch("/data?pag=0");` doesn't work

Reply:
oh I had to add it in the HTML too... ok fixed!